molecule/default/molecule-no-systemd.yml

```diff
@@ -0,0 +1,74 @@
+---
+# This molecule configuration file is suitable for testing Ansible
+# roles that _do not_ require SystemD.  If your Ansible role _does_
+# require SystemD then you should use molecule-with-systemd.yml
+# instead.
+#
+# Note that the molecule configuration file that is symlinked to
+# molecule.yml is the one that will be used.
+dependency:
+  name: galaxy
+driver:
+  name: docker
+lint: |
+  set -e
+  yamllint .
+  ansible-lint
+  flake8
+platforms:
+  # - name: amazonlinux2
+  #   image: amazonlinux:2
+  - name: debian9
+    image: debian:stretch-slim
+    dockerfile: Dockerfile_debian_9.j2
+  # - name: debian10
+  #   image: debian:buster-slim
```

Comment:
Also here?
```suggestion
  - name: debian10
    image: debian:buster-slim
```

Reply:
This PR just skeletonizes the repo.  #9 actually uses the new `pip` and `python` Ansible roles to expand the platforms.

Reply:
Ah, sorry- I missed that distinction, thanks!